test/unit/org/apache/cassandra/cql3/ast/Select.java

```diff
@@ -60,25 +60,26 @@ public class Select implements Statement
     public final Optional<OrderBy> orderBy;
     public final Optional<Value> limit;
     public final boolean allowFiltering;
-
+    public final boolean insertNewLine;
```

Comment:
please remove this from the state, we can add `toCQL` versions that allow overriding this

patch to fix here: https://github.com/dcapwell/cassandra/commit/1c8c34914d99633277940ebab1d12c16caba0898

Given these are not harry apis, and APIs I am maintaining, this is a blocking comment.

Reply:
Removed `insertNewLine`, replaced newlines with space in Harry compile code.

Reply:
I do not want you thinking you can not make improvements to the `ast` package... I was trying to express formatting decisions should be given to the logic that does the formatting rather than part of the object